docs/operate/get-started/supported-hardware.md

```diff
@@ -5,5 +5,367 @@ weight: 20
 layout: "docs"
 type: "docs"
 no_list: true
-description: "TODO"
+description: "Use supported hardware with your machine."
+modulescript: true
 ---
+
+Viam supports a wide variety of sensors, cameras, motors, robotic arms, and other physical hardware.
+Viam also supports various software services such as [data capture](/data-ai/get-started/capture-sync/), [computer vision](/data-ai/ai/create-dataset/), and [motion planning](/operate/mobility/move-arm/), designed to integrate seamlessly with the hardware driver modules.
+
+Any hardware that is not already supported by a Viam {{< glossary_tooltip term_id="module" text="module" >}} can be added into Viam's system of modular resources by [creating a new module](../other-hardware/) that provides a driver for the hardware.
```

Comment:
just because some of the cameras etc will stay builtin as far as I know... 
```suggestion
Any hardware that is not already supported by Viam can be added into Viam's system of modular resources by [creating a new module](../other-hardware/) that provides a driver for the hardware.
```

Reply:
I almost want people to think of every component as being supported by a module though since they might as well be, for understanding's sake? I think this change makes it sound more like there's a bigger difference between built in and modular.